stubs/mongodb.stub

```diff
@@ -0,0 +1,21 @@
+mongodb:
+    image: 'mongodb/mongodb-atlas-local:latest'
+    environment:
+        - MONGODB_INITDB_ROOT_USERNAME=${MONGODB_USERNAME:-}
+        - MONGODB_INITDB_ROOT_PASSWORD=${MONGODB_PASSWORD:-}
+    volumes:
+        - 'sail-mongodb:/data/db'
+    ports:
+        - '${FORWARD_MONGODB_PORT:-27017}:27017'
+    networks:
+        - sail
+    healthcheck:
+        test:
+            - CMD
+            - mongosh
+            - 'mongodb://localhost:27017/admin'
+            - '--username=${MONGODB_USERNAME:-}'
+            - '--password=${MONGODB_PASSWORD:-}'
```

Comment:
For the ping command, the credentials are not required. Is it more resilient to add them or to remove them?

Reply:
Removed the auth part. The password authentification is a command that is run after connection. So ping can always be called to check protocol connection.